### PR TITLE
Intro Tutorial - Move Code Playground

### PR DESCRIPTION
**Pull request by funDevProjects**
Initial commit of MCP Tutorial 01.

### CODE REVIEW

File: tutorials/move-code-playground-tutorial-01.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+
+
+#### Project Dependencies
+Clicking on the left navigation panel's 'Dependencies' icon reveals that the Move Code Playground comes pre-loaded with two dependencies:
+* AptosFramework - A set of modules specific to the Aptos blockchain.
+* MoveStdlib - The standard library of the Move language.
+
+*Note: When building your own projects, you may need to add these manually.*
+
+| Description       | Url                                       | Rev                           | Subdir |
```
Comment (borispovod) [inline tutorials/move-code-playground-tutorial-01.md:64]: Hey @funDevProjects 

The table is broken

File: tutorials/move-code-playground-tutorial-01.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+| Aptos Framework   | https://github.com/aptos-labs/aptos-core  | 367608ab5cf726039ff44bfdac1f2177210b1440     | aptos-move/framework/aptos-framework/ |
+| Move Std&nbsp;Lib | https://github.com/aptos-labs/aptos-core  | 367608ab5cf726039ff44bfdac1f2177210b1440     | aptos-move/framework/move-stdlib/     |
+
+#### Addresses
+Clicking on the left navigation panel's 'Addresses' icon reveals that the Move Code Playground comes pre-loaded with one Address:
+
+*Note: When building your own projects, you may need to add these manually.*
+
+| Name     | Address    | Description                                        |
+| ---------| -----------|----------------------------------------------------|
```
Comment (borispovod) [inline tutorials/move-code-playground-tutorial-01.md:75]: Probably that one too, double check please.

**Review by borispovod — CHANGES_REQUESTED** [review]

File: tutorials/move-code-playground-tutorial-01.md (hunk 1 above)
Comment (funDevProjects) [reply tutorials/move-code-playground-tutorial-01.md]: Fixed, sorry about that!

File: tutorials/move-code-playground-tutorial-01.md (hunk 2 above)
Comment (funDevProjects) [reply tutorials/move-code-playground-tutorial-01.md]: Updated and working now.

**Review by funDevProjects — COMMENTED** [review]
I added simple diagrams to improve clarity within the script demo sections.

**Review by borispovod — APPROVED** [review]

**Pull request merged**